go.mod

```diff
@@ -526,6 +532,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elast
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter => ./exporter/zipkinexporter
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/extension/asapauth => ./extension/asapauth
```

Comment:
This should be done on a separate PR; read the [contributing doc for new components](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#when-adding-a-new-component)

Reply:
Thanks, I've removed all the changes to the root `go.mod`. However, the build fails saying "asapauth/go.mod is not listed in any module set". Since I can't list it in the root go.mod, is there something I can do to make this green? Or is it fine to leave it like this.

Reply:
You need to add it on `versions.yaml` (we should add this step to the contributing guidelines, sorry, it's confusing)